osu.Game/Overlays/OnScreenDisplay.cs

```diff
@@ -8,34 +8,21 @@
 using osu.Framework.Configuration.Tracking;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
-using osu.Framework.Graphics.Shapes;
-using osu.Framework.Graphics.Sprites;
-using osu.Game.Graphics;
 using osuTK;
-using osuTK.Graphics;
-using osu.Framework.Extensions.Color4Extensions;
-using osu.Framework.Graphics.Effects;
 using osu.Framework.Graphics.Transforms;
 using osu.Framework.Threading;
 using osu.Game.Configuration;
-using osu.Game.Graphics.Sprites;
+using osu.Game.Overlays.OSD;
 
 namespace osu.Game.Overlays
 {
     public class OnScreenDisplay : Container
```

Comment:
needs xmldoc since you are overloading functionality.

"An on-screen display which automatically tracks and displays toast notifications for <see...TrackedSettings>. Can also display custom content via <see..Display>"

or something like this.